### ISSUE TITLE
`np.loadtxt` gives a warning for an empty file and returns `shape=(0, 1)`

### ISSUE THREAD

**Issue by seberg** [BIDS-numpy/npreadtext#53]
```python
from npreadtext import read, _loadtxt; from io import StringIO, BytesIO; import numpy as np

np.loadtxt(StringIO('''\n'''))
#<ipython-input-9-a3e642b8bbcd>:1: UserWarning: loadtxt: Empty input file: "<_io.StringIO object at 0x7efef5523a60>"
#  np.loadtxt(StringIO('''\n'''))
#Out[9]: array([], dtype=float64)
```
It also returns assumes a single column:
```
np.loadtxt(StringIO('''\n'''), ndmin=2)
# <UserWarning>
# array([], shape=(0, 1), dtype=float64)
```
returning a shape of `(0, 1)` and not `(0, 0)`

**Comment by seberg** [comment]
We need to add a test for this (and the warning for that matter).

I am starting to wonder if we should run a code coverage over the current Python code to write tests :/...

**Comment by seberg** [comment]
Actually, NumPy may have a test, I noticed yesterday that `read` just behaved differently from the `loadtxt` code.  There may be a point to that (i.e. moving the "reshape" to C) and using `(0, 0)` as the "correct" shape internally.

**Comment by seberg** [comment]
We have to put the warning back in (it is useful), it could make sense to even escelate to an Error.

Lets leave the shape handling as it was for these empty file cases.

**Comment by seberg** [comment]
@rossbar can I offload this to you for now as well? This should be done as pure Python post-processing.

(otherwise, ping me)

**Comment by rossbar** [comment]
Should be fixed by seberg/numpy#41
